src/shared/view/nunjucks/filters/charge-version-badge.js

```diff
@@ -1,16 +1,22 @@
+'use strict';
+
 const titleCase = require('title-case');
 
+const styles = {
+  current: 'completed',
+  draft: 'void',
+  approved: 'success',
+  replaced: 'inactive',
+  invalid: 'error',
+  review: 'warning'
```

Comment:
Do we need to handle the existing charge version status of superseded? Should this be handled in the water service?

Reply:
Resolved in d84c146c